docs/kernel_troubleshooting.rst

```diff
@@ -0,0 +1,183 @@
+Troubleshooting Kernel Updates
+==============================
+For security reasons, new versions of the Linux kernel running on your your 
+*Application* and *Monitor Servers* may be installed automatically as part of a 
+SecureDrop release. All kernel updates are tested extensively
+against :ref:`recommended hardware <Specific Hardware Recommendations>`. If
+things do go wrong (e.g., the server does not boot after a kernel update), 
+the following instructions will help you to roll back to the previous, 
+working kernel. You can then `report compatibility issues`_ to us so we can work
+together to resolve them as quickly as possible.
+
+First, you need to physically access each server. Power down the server
+(safely if possible), attach required peripherals (keyboard, monitor),
+and power the server back up. If you did not configure Google
+Authenticator for console access, you will need to use single user mode
```

Comment:
We refer to FreeOTP by default elsewhere in the documentation. A link to servers_2fa may help in case the person does not remember what it is about exactly because they chose Google Authenticator instead.

Reply:
per discussion during standup, this seems obsolete anyway. reworded accordingly.